packages/asset-server-plugin/src/types.ts

```diff
@@ -43,6 +43,24 @@ export interface ImageTransformPreset {
     mode: ImageTransformMode;
 }
 
+/**
+ * @description
+ * A configuration option for the Cache-Control header in the AssetServerPlugin asset response.
+ *
+ * @docsCategory AssetServerPlugin
+ */
+export type CacheConfig = {
+    /**
+     * The max-age=N response directive indicates that the response remains fresh until N seconds after the response is generated.
```

Comment:
You need to add `@description` tags to properties also so that the text shows up in the docs.